svg/struct/scripted/autofocus-attribute.svg

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml">
+    <title>Autofocus attribute</title>
+    <metadata>
+        <h:link rel="help" href="https://svgwg.org/svg2-draft/struct.html#autofocusattribute"/>
+    </metadata>
+    <a id="a1" href="#"></a>
+    <a id="a2" href="#"></a>
+    <h:script src="/resources/testharness.js"/>
+    <h:script src="/resources/testharnessreport.js"/>
+    <h:script src="/html/semantics/forms/autofocus/resources/utils.js"/>
+    <script><![CDATA[
+const SVG_NS = 'http://www.w3.org/2000/svg';
+
+promise_test(async t => {
+  let w = window.open('/common/blank.html');
+  await waitForLoad(w);
+  t.add_cleanup(() => { w.close(); });
+  const svg_a = w.document.createElementNS(SVG_NS, 'a');
+  svg_a.setAttribute('href', '#');
+  svg_a.setAttribute('autofocus', 'autofocus');
+  w.document.body.appendChild(svg_a);
```

Comment:
Adding an SVG `a` element directly to a `document.body` in a blank HTML file isn't correct. Since it's already been defined as an SVG-namespaced link, it shouldn't render outside of an SVG context.

Is there a reason all the tests are being done in a new document instead of this one containing the tests? To be sure there has been no user interaction that overrides the autofocus application?

Could we make that blank document an SVG file? Alternatively, you'd need to create an SVG element inside the external HTML file.

Reply:
I added blank.svg to this directory, and updated test cases to use it.  I wanted to have multiple test cases in a single file rather than adding SVG files for each of test cases because in general the number of test files affects test performance significantly.

Reply:
Great, that should handle it. Looks like you haven't pushed your edits yet, though?

Reply:
Right.  I pushed just now.